package.json

```diff
@@ -26,21 +26,21 @@
   },
   "license": " BSD 3-Clause License",
   "devDependencies": {
-    "jest-cli": "21.2.1",
-    "nodemon": "1.12.1",
+    "jest-cli": "22.0.4",
```

Comment:
@amandeepmittal possible breaking change here 🙃

Reply:
You want me to revert back the PR? Or is everything fine with it? I thought you meant that you have updated dependencies.